src/entity/Source.ts

```diff
@@ -109,6 +111,12 @@ export class Source {
   @Index('IDX_source_flags_featured', { synchronize: false })
   flags: SourceFlagsPublic;
 
+  @Column({ type: 'text', nullable: true, default: null })
```

Comment:
You don't need default null if nullable is true

Reply:
Yep. Overkill. Was doing trial and error if the migration would by adding one at a time. Lol.